plutus-playground/plutus-playground-client/nixpkgs-src.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
use the nixpkgs src from the main project, and get rid of this file.

Reply:
This is needed for now for some purescript issues, we will get rid of it with patches or something before merging.

Reply:
Please fix.